autogpt/memory/local.py

```diff
@@ -7,10 +7,12 @@
 import numpy as np
 import orjson
 
+from autogpt.config import Config
 from autogpt.llm_utils import create_embedding_with_ada
 from autogpt.memory.base import MemoryProviderSingleton
 
-EMBED_DIM = 1536
+CFG = Config()
```

Comment:
Config loading process was chnaged. Update from master and use the new method CC @collijk

Reply:
@ntindle `local.py` has `EMBED_DIM = 1536` hardcoded outside of any class scope at top of the file. It is only being used in another non-class function:
```python
def create_default_embeddings():
    return np.zeros((0, EMBED_DIM)).astype(np.float32)
```

Reply:
Ok, done.
In the last commit i had to keep the hardcoded default value of 1536 because CacheContent uses dataclasses and needs a default value for his definition, however that value is't actually used, because that class is always initialized by passing embedded_dim from config now.

I tried with both GPT3.5 and Vicuna, seems ok